src/gt4py/next/program_processors/runners/dace_fieldview/transformations/auto_opt.py

```diff
@@ -48,18 +47,33 @@ def gt_simplify(
     will be forwarded to DaCe, i.e. `GT_SIMPLIFY_DEFAULT_SKIP_SET` are not
     added automatically.
 
+    Passes that are replaced:
+    - Instead of `InlineSDFGs` the function will run `gt_inline_nested_sdfg()`.
+
     Args:
         sdfg: The SDFG to optimize.
         validate: Perform validation after the pass has run.
         validate_all: Perform extensive validation.
         skip: List of simplify passes that should not be applied, defaults
             to `GT_SIMPLIFY_DEFAULT_SKIP_SET`.
     """
-    return dace_passes_simplify.SimplifyPass(
+    if skip is None:
```

Comment:
Probably we should remove the `Optional` type, because anyway we have a default skip set. If instead we have to keep `Optional`, and I see then the need for this if-statement, then we can also move back the default set here.

Reply:
I see your point, but keeping optional would allow for `skip=None` to tell the transformation to not skip anything.
Otherwise we would have to write `skip=[]` or `skip=set()` which is more cryptic. But I removed it, what do you think?
@edopao

Reply:
@edopao I made the changes we discussed.